Quadriga/src/main/java/edu/asu/spring/quadriga/dto/WorkspaceDTO.java

```diff
@@ -21,7 +21,6 @@
 import javax.persistence.OneToMany;
 import javax.persistence.OneToOne;
 import javax.persistence.Table;
-import javax.xml.bind.annotation.XmlTransient;
```

Comment:
don't delete just the imports; you also need to delete where they are used. eclipse should have given you an error if you do that.